cloud/deployment/deployment.go

```diff
@@ -274,105 +291,275 @@ func Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeplo
 	}
 
 	// select and validate cluster
-	clusterID, err = useSharedClusterOrSelectDedicatedCluster(cloudProvider, region, c.OrganizationShortName, clusterID, clusterType, coreClient)
-	if err != nil {
-		return err
-	}
-
-	scheduler := astro.Scheduler{
-		AU:       schedulerAU,
-		Replicas: schedulerReplicas,
-	}
-
-	spec := astro.DeploymentCreateSpec{
-		Executor:  executor,
-		Scheduler: scheduler,
+	if !IsDeploymentStandard(deploymentType) {
+		clusterID, err = selectCluster(clusterID, c.Organization, corePlatformClient)
+		if err != nil {
+			return err
+		}
 	}
 
-	if dagDeploy == "enable" { //nolint: goconst
+	if dagDeploy == enable { //nolint: goconst
 		dagDeployEnabled = true
 	} else {
 		dagDeployEnabled = false
 	}
-
-	createInput := &astro.CreateDeploymentInput{
-		WorkspaceID:           workspaceID,
-		ClusterID:             clusterID,
-		Label:                 label,
-		Description:           description,
-		DagDeployEnabled:      dagDeployEnabled,
-		RuntimeReleaseVersion: runtimeVersion,
-		DeploymentSpec:        spec,
-		APIKeyOnlyDeployments: *enforceCD,
+	createDeploymentRequest := astroplatformcore.CreateDeploymentRequest{}
+	// build standard input
```

Comment:
is this comment correct since in the next line, we check if it is a standard deployment or dedicated deployment

Reply:
ahh i see, we can prob just move that comment down

Reply:
I moved it down